### PR TITLE
Cleanup Ansible style

### PR DESCRIPTION
**Pull request by SuperQ**
* Use yaml not old `=` syntax.
* Use `ansible_managed | comment` syntax.
* Fix up sysctl activation to use handler, not fact trigger.

Signed-off-by: Ben Kochie <superq@gmail.com>

### CODE REVIEW

File: ansible/playbooks/roles/common/handlers/main.yml [hunk 1]
```diff
@@ -10,3 +10,7 @@
     daemon_reload: true
     name: sshd
     state: reloaded
+
+- name: reload sysctl
+  become: true
```
Comment (krokodilerian) [inline ansible/playbooks/roles/common/handlers/main.yml]: is this needed? we usually run as root, and not sure how well sudo works...

**Comment by paulfantom** [comment]
Since sysctl template is just iterating over `sysctl-parameters` map, it might be better to switch to use `sysctl` ansible module. This way reload handler is not needed anymore.

File: ansible/playbooks/roles/common/handlers/main.yml (hunk 1 above)
Comment (SuperQ) [reply ansible/playbooks/roles/common/handlers/main.yml]: Yea, I guess we can skip it.

**Comment by SuperQ** [comment]
@paulfantom Downside to `sysctl` module is it does each sysctl one at a time. But, it does validate each sysctl. I found this bug:

```
root@h1302-cam:/proc/sys# sysctl vt.global_cursor_default=0
sysctl: cannot stat /proc/sys/vt/global_cursor_default: No such file or directory
```

**Comment by paulfantom** [comment]
> Downside to sysctl module is it does each sysctl one at a time. But, it does validate each sysctl.

Yes, that's a tradeoff. Personally I think an acceptable one as it changes a naive way of applying sysctl values into a more robust one, which can validate what is being done.

**Comment by SuperQ** [comment]
Done, added a commit to fix up the sysctl module use. Tested on `h1302-cam.video.fosdem.org`.

**Pull request merged**